helm-charts/orchestrator/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.2.0-rc23
+version: 1.3.0-rc1
```

Comment:
Can you also update the version in the Makefile?

Reply:
Done it in the Dockerfiles as well